examples/kubernetes/manifests.yml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: traefik-ondemand-service
+  namespace: kube-system
+  labels:
+    app: traefik-ondemand-service
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: traefik-ondemand-service
+  template:
+    metadata:
+      labels:
+        app: traefik-ondemand-service
+    spec:
+      serviceAccountName: traefik-ondemand-service
+      serviceAccount: traefik-ondemand-service
+      containers:
+      - name: traefik-ondemand-service
+        #image: gchr.io/acouvreur/traefik-ondemand-service
+        image: docker.io/mschneider82/traefik-ondemand-service
```

Comment:
we need to change this once upstream support is merged

Reply:
We should also fix the version. The version will be updated every time a release is made to make sure this stays compatible.
I want to avoid using :latest tag

Reply:
done